src/main/asciidoc/index.adoc

```diff
@@ -199,6 +199,25 @@ Vert.x will also provide stack traces to pinpoint exactly where the blocking is
 If you want to turn off these warnings or change the settings, you can do that in the
 {@link io.vertx.core.VertxOptions} object before creating the Vertx object.
 
+== Async results
+
+Vert.x 4 use futures to model asynchronous results.
+
+Any asynchronous method returns a {@link io.vertx.core.Future} object for the result of the call:
+a _success_ or a _failure_.
+
+You cannot interact directly with a future instance, instead you need to set a handler that will be called when the future
```

Comment:
It's not that you can't interact with a Future. It's the result you can't directly interact with ;-)